examples/custom-log-format/README.md

```diff
@@ -6,12 +6,9 @@ This example lets you set the log-format for NGINX using the configmap reosurce
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: nginx-config
```

Comment:
the name we use in the default  `manifests` is `nginx-config` (see https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/common/nginx-config.yaml#L4 ). could we use the same name for consistency? I also suggest adding the `nginx-ingress` namespace. Also, please see a similar example with the configmap -- https://github.com/nginxinc/kubernetes-ingress/tree/master/examples/custom-templates